projectcard/projectcard.py

```diff
@@ -21,8 +21,8 @@
     "roadway_deletion",
     "transit_property_change",
     "transit_routing_change",
+    "transit_route_addition",
```

Comment:
I am going bananas b/c I pushed this change to `add-new-transit-route` but it doesn't seem to have gone through so not sure why it is showing a diff!?  (a few others too).  Arg.